### DISCUSSION TITLE
Manually purging or deleting messages in queue

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#2143] [Help with using Postal]
We had a snafu that inadvertently sent a bunch of email out via Postal. Problem is its already in queue, like 20k messages. I stopped Postal to prevent the delivery, but as soon as I start Postal back up it wants to send them.

I tried to manually delete the messages out of the database, but when I start Postal, it still says 20k are in queue and looks like its still trying to deliver them.

How can I an override this and delete everything from queue?

**Comment by zdeneksvarc** [comment]
I don't know the internals, but you'll probably have to purge RabitMQ front, which has persistent volume.

**Reply by willpower232** [reply]
if the right messages have been removed from the database then I think the worker will deal with it and not send the queued message as it won't be able to get the rest of the information about the message
